src/app/lightning/payment-status-checker.ts

```diff
@@ -14,7 +15,7 @@ export const PaymentStatusChecker = ({ paymentRequest }) => {
     invoiceIsPaid: async (): Promise<boolean | RepositoryError> => {
       const ledger = LedgerService()
       const recorded = await ledger.isLnTxRecorded(paymentHash)
-      if (recorded instanceof RepositoryError) return recorded
+      if (recorded instanceof LedgerServiceError) return recorded
```

Comment:
this is one of the first implementations (is not consistent with actual pattern). If we are not doing error wrapping then you should use only `Error`

```suggestion
      if (recorded instanceof Error) return recorded
```